blockbase/block-templates/search.html

```diff
@@ -1,4 +1,4 @@
-<!-- wp:template-part {"slug":"header","tagName":"header","layout":{"inherit":true}} /-->
+<!-- wp:template-part {"slug":"header","tagName":"header"} /-->
```

Comment:
This I think was an unrelated change, but was discovered while scrutinizing the layout properties in the templates.  This particular change just makes the markup of all usages of the header template part the same.